incubator/hoverfly/values.yaml

```diff
@@ -0,0 +1,21 @@
+# Default values for hoverfly.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: docker.io/spectolabs/hoverfly
+  tag: v1.0.0-rc.2
```

Comment:
Can you please update to follow the format documented at https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#configuration

Reply:
Thank you for your review. I've added the `image.pullPolicy` as per the guideline. Is it good enough?